terraform/environments/production/relays.tf

```diff
@@ -8,95 +8,251 @@ module "relays" {
+      replicas   = 1
+      zones      = ["asia-east1-a"]
     }
-
-    "australia-southeast1" = {
+    "asia-east2" = {
       cidr_range = "10.131.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["australia-southeast1-c"]
+      replicas   = 1
+      zones      = ["asia-east2-a"]
     }
-
-    "europe-west1" = {
+    "asia-northeast1" = {
       cidr_range = "10.132.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["europe-west1-d"]
+      replicas   = 1
+      zones      = ["asia-northeast1-a"]
     }
-
-    # "me-central1" = {
-    #   cidr_range = "10.133.0.0/24"
-    #   type       = "e2-micro"
-    #   replicas   = 2
-    #   zones      ["me-central1-a"]
-    # }
-
-    "southamerica-east1" = {
+    "asia-northeast2" = {
+      cidr_range = "10.133.0.0/24"
+      type       = "e2-micro"
+      replicas   = 1
+      zones      = ["asia-northeast2-a"]
+    }
+    "asia-northeast3" = {
       cidr_range = "10.134.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["southamerica-east1-c"]
+      replicas   = 1
+      zones      = ["asia-northeast3-a"]
     }
-
-    "us-central1" = {
+    "asia-south1" = {
       cidr_range = "10.135.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["us-central1-f"]
+      replicas   = 1
+      zones      = ["asia-south1-a"]
     }
-
-    "us-east1" = {
+    "asia-south2" = {
       cidr_range = "10.136.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["us-east1-c"]
+      replicas   = 1
+      zones      = ["asia-south2-a"]
     }
-
-    "us-west2" = {
+    "asia-southeast1" = {
       cidr_range = "10.137.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["us-west2-b"]
+      replicas   = 1
+      zones      = ["asia-southeast1-a"]
     }
-
-    "europe-central2" = {
+    "asia-southeast2" = {
       cidr_range = "10.138.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["europe-central2-c"]
+      replicas   = 1
+      zones      = ["asia-southeast2-a"]
     }
-
-    "europe-north1" = {
+    "australia-southeast1" = {
       cidr_range = "10.139.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["europe-north1-c"]
+      replicas   = 1
+      zones      = ["australia-southeast1-a"]
     }
-
-    "europe-west2" = {
+    "australia-southeast2" = {
       cidr_range = "10.140.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["europe-west2-c"]
+      replicas   = 1
+      zones      = ["australia-southeast2-a"]
     }
-
-    "us-east4" = {
+    "europe-central2" = {
       cidr_range = "10.141.0.0/24"
       type       = "e2-micro"
-      replicas   = 2
-      zones      = ["us-east4-c"]
+      replicas   = 1
+      zones      = ["europe-central2-a"]
+    }
+    "europe-north1" = {
+      cidr_range = "10.142.0.0/24"
```

Comment:
I know this is just following the pattern that was already established, but was there a particular reason we're eating up so much of our `10.x.x.x` CIDR space when each of these networks is a `/24`?

Couldn't all of these be something like:
```
10.129.0.0/24
10.129.1.0/24
10.129.2.0/24
...
```

Reply:
I wondered the same thing but decided to keep the changes minimal unless there was an issue with the numbering.

I think these can be easily changed in the future if it does create an issue.

Reply:
👍